configuration.py

```diff
@@ -195,6 +202,7 @@ class Parameter:
     LONGITUDE = 'longitude'
     AUTOMATIC_LOCATION_DETECTION = 'automatic_location_detection'
     TEMPERATURE_SCALE = 'temperature_scale'
+    WIND_SCALE = 'wind_scale'
```

Comment:
If you add a new parameter, you need to modify `def __apply_change_log(self):` so that if the user upgrade, the program does not fail because cannot read `wind_scale` property from config file